python/ml4ir/base/config/parse_args.py

```diff
@@ -59,24 +65,25 @@ def define_args(self):
         self.add_argument(
             "--loss_key",
             type=str,
-            default=None,
-            help="Loss to optimize. Has to be one of the losses in LossKey under ml4ir/config/keys.py",
+            choices=RankingLoss.get_all_keys() + ClassificationLoss.get_all_keys(),
+            help="Loss to optimize."
         )
 
         self.add_argument(
             "--metrics_keys",
             type=str,
+            nargs="+",
```

Comment:
@balikasg I don't think this will work on MLS/sip-pycli.

Reply:
Works fine on MLS @lastmansleeping , see SearchAssistantModeling/configs/train_ner.yaml (line 13)

Reply:
I tested with EP too.